pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchgit, fetchFromGitHub, cmake, pkgconfig, git, python3,
+  python3Packages, glslang, spirv-tools, x11, libxcb, wayland }:
+
+assert stdenv.system == "x86_64-linux";
+
+let
+  version = "1.0.26.0";
+  src = fetchFromGitHub {
+    owner = "KhronosGroup";
+    repo = "Vulkan-LoaderAndValidationLayers";
+    rev = "sdk-${version}";
+    sha256 = "157m746hc76xrxd3qq0f44f5dy7pjbz8cx74ykqrlbc7rmpjpk58";
+  };
+  getRev = name: builtins.substring 0 40 (builtins.readFile "${src}/${name}_revision");
```

Comment:
There's a CI error here

`error: cannot read ‘/nix/store/mll9kdy5pi9wirn088zpkh0hj2wk742y-Vulkan-LoaderAndValidationLayers-sdk-1.0.26.0-src/spirv-tools_revision’, since path ‘/nix/store/i5y4wagj1zjbd490ib4qls7zff6adpm2-Vulkan-LoaderAndValidationLayers-sdk-1.0.26.0-src.drv’ is not valid, at /home/travis/build/NixOS/nixpkgs/pkgs/development/libraries/vulkan-loader/default.nix:14:43`